src/a11y-dialog.ts

```diff
@@ -74,6 +77,16 @@ export default class A11yDialog {
     // If the event was prevented, do not continue with the normal behavior
     if (showEvent.defaultPrevented) return this
 
+    // When opening the dialog, create a new `CloseWatcher` instance, and listen
+    // for a close event to call our `.hide(..)` method and nuking the close
+    // watcher once it’s been consumed
+    if (typeof CloseWatcher !== 'undefined') {
```

Comment:
I think this should be enough to avoid runtime errors if the browser does not support the `CloseWatcher` API.